content/posts/building_jnigen.md

```diff
@@ -0,0 +1,262 @@
+---
+title: "jnigen - Building JNI bindings generator for Dart Language"
+type: "post"
+date: 2023-06-29T07:49:21+05:30
+draft: true
+tags: [dart, flutter, jnigen]
+showTableOfContents: true
+---
+>> Caine: Then why bother?
+>>
+>> John: Maybe I am wrong.
+>>
+> John Wick 4, (2023)
+
+It's been a long overdue to write a post about my past year's Google Summer of Code (GSoC) project. `jnigen` is an experimental bindings generator which aims to provide Java interoperability for Dart. It works by generating wrappers which call JNI through Dart's FFI (Foreign Function Interface).
+
+I developed the initial versions of this package under the guidance of Daco Harkes and Liam Appelbe from Dart language team. Hossein Yousefi from Dart team is developing the project further, adding many features such as Generics and Kotlin language support.
```

Comment:
We're part of the Dart team, but not necessarily the Dart language team. (The language team within the Dart team are the people who decide about what language features to put into dart: https://github.com/dart-lang/language.)